QuizPrototype/Data/Models/Test.cs

```diff
@@ -16,5 +16,6 @@ public class Test
         public string subSubTheme { get; set; }
```

Comment:
```suggestion
        public string SubSubTheme { get; set; }
```
В случае автосвойств, нужно называть их с большой буквы